src/T2DMap.cpp

```diff
@@ -695,7 +694,6 @@ void T2DMap::paintEvent( QPaintEvent * e )
     py = oy*ty+_ry;
 
     TArea * pArea = pAID;
-    if( ! pArea ) return;
```

Comment:
Is there any case where `pArea` could be a `nullptr`?  The program logic is a bit twisty in this region...!